core/src/main/java/io/grpc/internal/DnsNameResolver.java

```diff
@@ -82,9 +84,13 @@
   @GuardedBy("this")
   private Listener listener;
 
-  DnsNameResolver(@Nullable String nsAuthority, String name, Attributes params,
+  DnsNameResolver(
+      @Nullable String nsAuthority,
+      String name,
+      Attributes params,
       Resource<ScheduledExecutorService> timerServiceResource,
-      Resource<ExecutorService> executorResource) {
+      Resource<ExecutorService> executorResource,
+      boolean balancerLookup) {
```

Comment:
There will be only one resolver that does DNS, and it will optionally do SRV.  There should not be two.